### DISCUSSION TITLE
European AppStore availabilty

### DISCUSSION THREAD

**Discussion by IpsmLorem** [MobiusSync/MobiusSync#16] [Q&A] [answered]
Hi.

Do you plan to release MobiusSync to all european Countries App Store ?

Thank you.

**Comment by SimonPickup** [comment]
I understand it is available in all country App Stores except France, because it has specific encryption documentation requirements.

Are you searching in the French app store?

**Reply by PaulPichaureau** [reply]
Hello. I've searched in the French appstore, and MobiusSync is not available. I cannot change my region due to subscriptions (Apple Music, Apple TV).

**Reply by SimonPickup** [reply]
Having done some more research, the process for French approval for approving encryption seems quite onerous and would take many months. Not something we can invest in at this stage. And even when we can, it sounds like at least three months to obtain approval. Sorry to our French supporters.

**Comment by SimonPickup** [comment, marked as answer]
Now Issue #27.

**Comment by pedrosimao** [comment]
I also need it in France appstore! Thanks for looking into the issue.
